07_deploy_masters.sh

```diff
@@ -50,14 +50,16 @@ popd
 
 echo "Master nodes active"
 
-NUM_LEASES=$(sudo virsh net-dhcp-leases baremetal | grep master | wc -l)
-while [ "$NUM_LEASES" -ne 3 ]; do
-  sleep 10
+if [ "$OOOQ_SETUP" == 'y' ]; then
```

Comment:
I wonder if we can find a cleaner way to do this, maybe we ping the hosts by name instead which would work in both cases?

Reply:
Yes, we can leverage the DNS entries. I sent another PR to address it - https://github.com/openshift-metalkube/dev-scripts/pull/232